crates/diesel_models/src/payment_intent.rs

```diff
@@ -59,10 +59,11 @@ pub struct PaymentIntent {
     pub request_external_three_ds_authentication: Option<bool>,
     pub charges: Option<pii::SecretSerdeValue>,
     pub frm_metadata: Option<pii::SecretSerdeValue>,
+    pub guest_customer_details: Option<Encryption>,
```

Comment:
can you change this to customer_details? Since we will be storing the information for all the customers